view/frontend/web/autocomplete.js

```diff
@@ -81,7 +81,7 @@ requirejs(['algoliaBundle'], function(algoliaBundle) {
 				options.templates.footer = '<div class="footer_algolia"><a href="https://www.algolia.com/?utm_source=magento&utm_medium=link&utm_campaign=magento_autocompletion_menu" title="Search by Algolia" target="_blank"><img src="' +algoliaConfig.urls.logo + '"  alt="Search by Algolia" /></a></div>';
 			}
 			
-			sources = algolia.triggerHooks('beforeAutocompleteSources', sources, algolia_client);
+			sources = algolia.triggerHooks('beforeAutocompleteSources', sources, algolia_client, algoliaBundle);
```

Comment:
Why the `algoliaBundle` is needed there, @damcou ?

Reply:
I added it cause I couldn't complete the "add an external source" tutorial without using it : https://github.com/damcou/algoliasearch-magento2-external-source/blob/master/view/frontend/web/hooks.js#L10-L30

I don't really if there's another solution.

Reply:
ok, makes sense :) LGTM then!